net.longturn.freeciv21.yaml

```
@@ -0,0 +1,74 @@
+##########
+# Freeciv21 - net.longturn.freeciv21.yaml
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: James Robertson <jwrober@gmail.com>
+#
+# Build Steps:
+#   - (once) sudo apt install flatpak-builder
+#   - (once) sudo flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
+#   - flatpak-builder --install-deps-from=flathub --force-clean build/flatpak dist/net.longturn.freeciv21.yaml
+#   - flatpak-builder --user --install --force-clean build/flatpak dist/net.longturn.freeciv21.yaml
+#   - flatpak run net.longturn.freeciv21
+#   - flatpak remove net.longturn.freeciv21
+#
+##########
+
+app-id: net.longturn.freeciv21
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=home
+  - --filesystem=~/.local/share/freeciv21
+
+command: freeciv21-client
+modules:
+  # Get Lua 5.3 as a dependency not included in the org.kde.Platform package.
+  - name: Lua
```

```suggestion
  - name: Lua
  - cleanup:
      - '*'
```

I think yu want to remove it all.

---

Thanks. This does remove it.